CMakeLists.txt

```diff
@@ -269,6 +269,11 @@ if (PRECICE_FEATURE_GINKGO_MAPPING)
 
   # 2. Check if Ginkgo was built with HIP and if HIP is available on the system
   if(PRECICE_WITH_HIP)
+    include(CheckLanguage)
```

Comment:
Is it safe to include this multiple times?

Reply:
Despite the fact that executing both if branches seems like a very rare case, it is safe AFAIK.